contracts/contracts/lib/LibGateway.sol

```diff
@@ -423,7 +423,7 @@ library LibGateway {
                 sendReceipt(crossMsg, OutcomeType.SystemErr, abi.encodeWithSelector(InvalidXnetMessage.selector, InvalidXnetMessageReason.Nonce));
                 return;
             }
-            subnet.appliedBottomUpNonce += 1;
```

Comment:
Given that this is purely stylistic, I would use a post-increment. It's visually more common.

Reply:
@cryptoAtwill Can't we do `appliedBottomUpNonce++`?

Reply:
I think `++appliedBottomUpNonce` is `appliedBottomUpNonce += 1` but `appliedBottomUpNonce++` is actually:
```
i = appliedBottomUpNonce;
appliedBottomUpNonce = appliedBottomUpNonce + 1;
i
```
it's slightly more gas consuming.